app.py

```diff
@@ -1,23 +1,56 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import os
+import requests
+from todo_item import ToDoItem
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    items = session.get_items()
+    board_id = os.getenv('BOARD_ID')
+    api_key = os.getenv('TRELLO_API_KEY')
+    api_secret = os.getenv('TRELLO_API_TOKEN')
```

Comment:
You could try to reduce code duplication by creating a function that fetches the key and token and returns a dictionary `{...}`.